### DISCUSSION TITLE
What is that?

### DISCUSSION THREAD

**Discussion by ArtemTubeGithub** [git-lfs/git-lfs#5755] [Q&A]
My Git LFS doesn't wanna be installed on my PC, although Github Desktop is installed.

https://github.com/git-lfs/git-lfs/assets/160143407/4a414a1b-0fa0-4886-8910-455fe32138b2

**Comment by chrisd8088** [comment]
Hey, I'm sorry you're having trouble.  It looks like you're using GitHub Desktop on a Windows system, but you don't provide any details as to the versions of either, or if you have Git for Windows installed, etc.

I'm afraid we'll need more details in order to try to help, but to be honest, you'll likely get a better answer from the GitHub Desktop support team, as this project is just for the open-source Git LFS client, so we can't speak to how it's integrated into other products.

**Reply by ArtemTubeGithub** [reply]
Version 3.4.2-beta3 (x64), and it's installed. This error appears even when GitHub Desktop has been closed.

**Reply by bk2204** [reply]
You need Git for Windows installed in `PATH` if you want to use Git LFS.  If it's not installed in `C:\Program Files\Git`, then you'll get a confirmation prompt, but it should still be installable.

GitHub Desktop installs Git into a different location and should ship Git LFS with it if you want to use that version, but if you want to use GitHub Desktop's Git version and a different Git LFS, then you need to add the appropriate entries into `PATH`.
